owned-manifests/clusterapi-manager-controllers.yaml

```diff
@@ -59,6 +59,13 @@ spec:
         args:
           - --logtostderr=true
           - --v=3
+      - name: nodelink-controller
+        image: {{ .Controller }}
```

Comment:
Oh, we need to use the mao docker image. The `nodelink-controller` is built as part of it.